pyanaconda/modules/storage/checker/utils.py

```diff
@@ -53,7 +53,11 @@ def verify_root(storage, constraints, report_error, report_warning):
                        "which is required for installation of %s"
                        " to continue.") % (get_product_name(),))
 
-    if root and root.format.exists and root.format.mountable and root.format.mountpoint == "/":
+    if (root
+        and root.format.exists
+        and root.format.mountable
+        and root.format.mountpoint == "/"
+        and (root.format.free != root.format.total)):
```

Comment:
I am not sure this will work, even newly created filesystems are not "empty", for example ext4 reserves 5 % of free space on the filesystem by default so filesystem free space will always be less than filesystem size. This is a freshly created 1 GiB Ext4 with just 973.14 MiB of free blocks:

```
# dumpe2fs /dev/sdd2
...
Block count:              262144
Reserved block count:     13107
Overhead clusters:        12949
Free blocks:              249125
Free inodes:              65524
First block:              0
Block size:               4096
...
```

Reply:
I see, yes I just wanted to try it out and pushed a PR instead of building locally.